x-pack/filebeat/module/cisco/ftd/test/dns.log-expected.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "@timestamp": "2019-08-26T21:11:03.000-02:00",
+        "@timestamp": "2019-08-26T23:11:03.000Z",
```

Comment:
Note that this change is actually a bugfix; Z is UTC, but was previously parsed in local.